cmd/controller/ctrl.go

```diff
@@ -84,6 +90,17 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 		glog.Error("Unable to push disk objects to etcd : ", err)
 		return err
 	}
+	//This infinite loop will help to get event details send by udev-event module
+	//event details send using channel.
+	for true {
```

Comment:
if receiving values through channels is the moto, pls use golang select statement